src/ranged.cpp

```diff
@@ -71,15 +71,98 @@ size_t blood_trail_len( int damage )
     return 0;
 }
 
+// projectile_attack handles a single shot aimed at
```

Comment:
What is this wordwrapped to? col 100 would be reasonable. Less vertical paging is a good thing for code clarity.